utils/lxd/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (C) 2019 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lxd
+PKG_VERSION:=3.11
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Florian Eckert <fe@dev.tdt.de>
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://linuxcontainers.org/downloads/lxd/
+PKG_HASH:=5d38ca401aedbba867f2b8b4cb491efe85047dd0729f22b31ae2feef21cfbf77
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+
+GO_PKG:=github.com/lxc/lxd
+GO_PKG_BUILD_PKG:=github.com/lxc/lxd/lxd/
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/../feeds/packages/lang/golang/golang-package.mk
+
+TAR_CMD:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components 1 $(TAR_OPTIONS)
```

Comment:
serious?  that's an unusual approach. normally people just edit hte PKG_SOURCE directories appropriately.  I knwo it's "equivalent" but this is much more intrusively reaching into the build guts.